tests/t097_dump_basic.py

```diff
@@ -14,7 +14,7 @@ def __init__(self):
 uftrace file header: endian        = 1 (little)
 uftrace file header: class         = 2 (64 bit)
 uftrace file header: features      = 0x263 (PLTHOOK | TASK_SESSION | SYM_REL_ADDR | MAX_STACK | AUTO_ARGS)
-uftrace file header: info          = 0xbff
+uftrace file header: info          = 0xbfd
```

Comment:
Hi @honggyukim 😄 

I think it'd be better to modify `fixup()` instead of expected output like below:
Because it is related to Build ID.

```patch
diff --git a/tests/t097_dump_basic.py b/tests/t097_dump_basic.py
index 595a251..1a975b9 100644
--- a/tests/t097_dump_basic.py
+++ b/tests/t097_dump_basic.py
@@ -52,5 +52,5 @@ reading 5231.dat
             result = result.replace("2 (64 bit)", "1 (32 bit)")
         p = sp.Popen(['file', 't-' + self.name], stdout=sp.PIPE)
         if 'BuildID' not in p.communicate()[0].decode(errors='ignore'):
-            result = result.replace("0x3ff", "0x3fd")
+            result = result.replace("0xbff", "0xbfd")
         return result
```
First of all, we can check the change history(`0x3ff` -> `0xbff`) on 5e0abb1
(This is related to added new info bit `RECORD_DATE`)

And the difference between `0xbff` and `0xbfd` depends on whether a build id exists like below:
`0xbfd` : 0000 0000 1011 1111 1101
`0xbff` :  0000 0000 1011 1111 1111
So I think it'd be better to change `fixup()`.
(Of cource, you can check the code about info bit on https://github.com/namhyung/uftrace/blob/master/cmd-info.c#L835 )

You can also see the change of `fixup()` due to this situation on e01ae7d

Reply:
Thanks. I've done it in bd25cb1.